### ISSUE TITLE
44219300: Interface Builder should show the Accessibility pane for all objects implementing UIAccessibilityIdentification

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20382]
#### Description

Summary:
When selecting some UI elements in a storyboard (say, a UITabBarItem), the Accessibility pane is not shown, and one is forced to add values for accessibility properties in the "User Defined Runtime Attributes" pane, which is error-prone and brittle.

The Accessibility pane should be show for all element implementing UIAccessibilityIdentification.

Steps to Reproduce:
1. Open a .storyboard file with a UITabBarController
2. Click on one of the UITabBarItem

Expected Results:
In the Inspector pane on the right side of Xcode, the Accessibility pane is shown

Actual Results:
The Accessibility pane is not show, and the user is forced to use the User Defined Runtime Attributes or set the properties in code

Version:
10.0 beta 6 (10L232m)

Notes:
Reproduced in Xcode 9.4.1 (9F2000) and 10.0 beta 6 (10L232m)

-
Product Version: 10.0 beta 6 (10L232m)
Created: 2018-09-07T10:28:56.524240
Originated: 2018-09-07T12:28:00
Open Radar Link: http://www.openradar.me/44219300
